File: striver/S2L1/selection.py
size = int(input())
arr = [0]*size
for i in range(0,size):
    arr[i]=int(input())
def selection_sort(arr):
    for i in range(0,size):
        point = int(0) 
        for j in range(0,size-i):
            if arr[j]>arr[point]:
                point = j
        temp = arr[size-i-1] 
        arr[size-i-1] = arr[point]
        arr[point] = temp
        # for i in range(0,size):
        #     print(arr[i],end=" ")
        # print()
selection_sort(arr) 
for i in range(0,size):
        print(arr[i],end=" ")

File: striver/S2L1/selection.java
package S2L1;

import java.util.Scanner;

public class selection {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        for(int i=0;i<n;i++)
            arr[i] = sc.nextInt();
        ss(arr);
        for(int i=0;i<n;i++)
            System.out.print(arr[i]+" ");
    }
    public static void ss(int[] arr){
        for(int i=0;i<arr.length;i++){
            int point = 0;
            for(int j=0;j<arr.length-i;j++){
                if(arr[point]<arr[j])
                    point = j;
            }
            int temp = arr[point];
            arr[point] = arr[arr.length-1-i];
            arr[arr.length-1-i]=temp;
        }
    }
}
